x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -129,34 +129,48 @@ const ActionsComponent: React.FC<ActionProps> = ({
     }
   }, [dispatch, ecsData._id, timelineId, setGlobalFullScreen, setTimelineFullScreen]);
 
-  const entryLeader = useMemo(() => {
-    const { process } = ecsData;
-    const entryLeaderIds = process?.entry_leader?.entity_id;
-    if (entryLeaderIds !== undefined && entryLeaderIds.length > 0) {
-      return entryLeaderIds[0];
-    } else {
+  const sessionViewConfig = useMemo(() => {
+    const { process, _id, timestamp } = ecsData;
+    const sessionEntityId = process?.entry_leader?.entity_id?.[0];
+
+    if (sessionEntityId === undefined) {
       return null;
     }
-  }, [ecsData]);
+
+    const jumpToEntityId = process?.entity_id?.[0];
+    const investigatedAlertId = eventType === 'signal' ? _id : undefined;
+
+    let jumpToCursor = timestamp;
+    if (investigatedAlertId) {
+      jumpToCursor = ecsData.kibana?.alert.original_time?.[0];
```

Comment:
it can happen of `jumpToCursor` being re-assigned to `undefined`? if so, I think something like this could work instead:

```js
const jumpToCursor = investigatedAlertId && ecsData.kibana?.alert.original_time?.[0] || timestamp
```